### PR TITLE
remove GasNow dependency

### PR DESCRIPTION
**Pull request by onnovisser**
Remove GasNow dependency for gas prices and use `maxFee` and `maxPriorityFee` instead

### CODE REVIEW

File: onboarding/api/src/utils/tx-manager.ts [hunk 1]
```diff
@@ -62,12 +45,11 @@ class TransactionManager extends ethers.Signer {
       if (this.config.filterDuplicates && !increases && this.transactions[key]) {
         throw new Error(`Transaction ${key} already sent`)
       }
-
       this.transactions[key] = { request: transaction, resolve, reject }
       this.queue.push(key)
 
       if (this.queue.length === 1) {
-        console.log(`Processing ${key} immediately`)
+        console.log(`Procesisng ${key} immediately`)
```
Comment (offerijns) [inline onboarding/api/src/utils/tx-manager.ts:52]:
```suggestion
        console.log(`Processing ${key} immediately`)
```

File: onboarding/api/src/services/pool.service.ts [hunk 2]
```diff
@@ -20,10 +20,7 @@ export class PoolService {
   private pools: { [key: string]: Pool } = {}
 
   provider = new FastJsonRpcProvider(config.rpcUrl)
-  signer = new TransactionManager(new ethers.Wallet(config.signerPrivateKey), {
-    initialSpeed: 'fast',
-    increasedSpeed: 'rapid',
-  }).connect(this.provider)
+  signer = new TransactionManager(new ethers.Wallet(config.signerPrivateKey)).connect(this.provider)
```
Comment (offerijns) [inline onboarding/api/src/services/pool.service.ts:23]: Can you set the `initialPriorityFeePerGas` here to 4 already? I think that's worth it for whitelisting, which has some time urgency, and these transactions use a small amount of gas so the impact is relatively low.

Comment (offerijns) [reply onboarding/api/src/services/pool.service.ts:23]: Can you set `maxFeePerGas` to 600 for the onboarding API and 300 for the bot? A bit ridiculous, but we're now sometimes also paying this, and if gas is high for a day, we shouldn't stop closing epochs entirely.

**Review by offerijns — APPROVED** [review]

File: onboarding/api/src/services/pool.service.ts (hunk 2 above)
Comment (onnovisser) [reply onboarding/api/src/services/pool.service.ts]: updated

**Pull request merged**